pkgs/misc/tmux-plugins/default.nix

```
@@ -127,6 +127,24 @@ in rec {
     };
   };
 
+  dracula = mkDerivation rec {
+    pluginName = "dracula";
+    version = "1.0.1";
+    src = fetchFromGitHub {
+      owner = "dracula";
+      repo = "tmux";
+      rev = "cb1d420a6267c600a8ef0e7a3c00e2474b30ae99";
+      sha256 = "1hz2grgq6h9lb0n33rl12qjslnky6zs4r494fcb9h08vym9z9sj0";
+    };
+    meta = {
+      homepage = "https://draculatheme.com/tmux";
+      description = "A feature packaed Dracula theme for tmux!";
+      license = stdenv.lib.licenses.mit;
+      platforms = stdenv.lib.platforms.unix;
+      maintainers = with stdenv.lib.maintainers; [ ethancedwards8 ];
+    };
+  };
```

```suggestion
    meta = with stdenv.lib; {
      homepage = "https://draculatheme.com/tmux";
      description = "A feature packaed Dracula theme for tmux!";
      license = licenses.mit;
      platforms = platforms.unix;
      maintainers = with maintainers; [ ethancedwards8 ];
    };
  };
```

But won't block the merge.